src/ontology/config/prefixes.csv

```diff
@@ -225,7 +225,7 @@ CARO,http://purl.obolibrary.org/obo/CARO_
 NPO,http://purl.bioontology.org/ontology/npo#NPO_
 ICD10CM,http://purl.bioontology.org/ontology/ICD10CM/
 ICD10WHO,https://icd.who.int/browse10/2019/en#/
-OMIMPS,https://omim.org/phenotypicSeries/PS
+OMIMPS,https://www.omim.org/phenotypicSeries/PS
```

Comment:
### Source of the problem
For:
- #350 

Related:
- https://github.com/monarch-initiative/omim/issues/103

Unfortunately the presence of `www` in these OMIMPS URIs did indeed cause some confusion/problems deeper in the pipeline.

The primary cause of the labels not showing up was this line in `config/prefixes.csv`.

But to prevent further issues, I fixed every other reference where `www` was missing. There were 7 or so instances, hence why so many files are edited here.

I also edited #103 to remind me of all the changes that would need to be made to `mondo-ingest` whenever we eventually remove `www` from the URL.